src/rp/util/string.clj

```diff
@@ -28,3 +28,8 @@
   (some-> s
           parse-double*
           double))
+
+(defn non-blank-string?
+  "Returns true if x is a non-blank string"
+  [x] (and (string? x)
```

Comment:
It'd be more consistent with the rest of our code to insert a newline btwn. arg vector and fn body.